libs/causality/src/lib/CausalAnalysisDashboard/Controls/CausalAnalysisView/CausalAggregateView/CausalAggregateView.tsx

```diff
@@ -108,15 +108,30 @@ export class CausalAggregateView extends React.PureComponent<ICausalAggregateVie
 
   private getContinuousDescription(): string {
     if (this.context.dataset.task_type === "classification") {
-      return localization.CausalAnalysis.AggregateView.continuousDescription;
+      let positiveClass = "1";
```

Comment:
What if it's multiclass? Is "positive" even a thing then?

Reply:
Can change the variable name if you think it is somewhat misleading. We don't have multiclass for causal.

Reply:
changed the variable name.